### PR TITLE
feat: add bs delete volume recover&clone command

### PR DESCRIPTION
**Pull request by setcy**
### What problem does this PR solve?

add `bs delete volume recover` and `bs delete volume clone`  command

```bash
$ curve bs delete volume clone
+------+--------------------------------------+--------------------------------------+--------+--------+            
| USER |                 SRC                  |               TASK ID                |  FILE  | RESULT |            
+------+--------------------------------------+--------------------------------------+--------+--------+            
| root | a19b5e5e-b306-488f-8e6d-d87282c869cb | d26e27a8-fcbd-4f7a-adf8-53795217cbb0 | /root2 | fail   |            
+------+--------------------------------------+--------------------------------------+--------+--------+  
```

```bash
$ curve bs delete volume recover
+------+--------------------------------------+--------------------------------------+-------+--------+             
| USER |                 SRC                  |               TASK ID                | FILE  | RESULT |             
+------+--------------------------------------+--------------------------------------+-------+--------+             
| root | a19b5e5e-b306-488f-8e6d-d87282c869cb | 9dfa8699-a275-4891-8ec2-e447a0ccc77c | /root | fail   |             
+------+--------------------------------------+--------------------------------------+-------+--------+   
```

Issue Number: 
#2580
#2581

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by setcy** [comment]
cicheck

**Comment by setcy** [comment]
cicheck

**Comment by setcy** [comment]
cicheck

**Comment by lng2020** [comment]
I believe there is no problem with the logic of the code, but the printing form may need to be standardized. The implementation in your PR is to ask the user yes/no, and then print it line by line, which is different from the printing form of other commands(e.g. print a table with several rows). But how to print is not up to me, ask @Cyber-SiKu  for advice.

File: tools-v2/pkg/config/bs.go [hunk 1]
```diff
@@ -606,6 +618,22 @@ func AddBsChunkServerAddressSliceRequiredFlag(cmd *cobra.Command) {
...
+	AddBsStringOptionFlag(cmd, CURVEBS_SRC, "source")
+}
+
+func AddBsDestOptionFlag(cmd *cobra.Command) {
+	AddBsStringOptionFlag(cmd, CURVEBS_DEST, "destination")
+}
+
+func AddBsTaskIDOptionFlag(cmd *cobra.Command) {
+	AddBsStringOptionFlag(cmd, CURVEBS_TASKID, "task id")
+}
```
Comment (lng2020) [inline tools-v2/pkg/config/bs.go:631]: you may change this three flags to required

File: tools-v2/pkg/cli/command/curvebs/delete/volume/clone_or_recover.go [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+	} else {
+		fmt.Println("Please enter yes or no")
+		return false
+	}
+}
+
+func (c *CloneOrRecover) cleanCloneOrRecover(taskID, user string) {
+	if taskID == "" && user == "" {
+		return
+	}
```
Comment (lng2020) [inline tools-v2/pkg/cli/command/curvebs/delete/volume/clone_or_recover.go:212]: and then remove this check

File: tools-v2/pkg/cli/command/curvebs/delete/volume/clone_or_recover.go [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+	if askRes == "yes" {
+		return true
+	} else if askRes == "no" {
+		return false
+	} else {
+		fmt.Println("Please enter yes or no")
+		return false
+	}
+}
```
Comment (lng2020) [inline tools-v2/pkg/cli/command/curvebs/delete/volume/clone_or_recover.go:207]: **Just in my opinion**. Sometimes the old tool checks the user's operation, but usually, the tool-v2 doesn't do that.

**Review by lng2020 — COMMENTED** [review]
those are my personal opinions. You should better discuss this with someone with a merge permission.

**Comment by setcy** [comment]
cicheck

File: tools-v2/pkg/config/bs.go (hunk 1 above)
Comment (setcy) [reply tools-v2/pkg/config/bs.go:631]: The requirement of the old tool is that it only needs one flag of them. It seems too demanding to require all these flags.

**Comment by setcy** [comment]
cicheck

**Comment by setcy** [comment]
cicheck

**Comment by Cyber-SiKu** [comment]
add some output

**Comment by Cyber-SiKu** [comment]
Please add the usage of the command in the readme

**Review by Cyber-SiKu — APPROVED** [review]

**Review by caoxianfei1 — APPROVED** [review]

**Pull request merged**